File: app/src/main/java/com/simon/harmonichackernews/data/CommentsScrollProgress.java
package com.simon.harmonichackernews.data;

import java.util.HashSet;

public class CommentsScrollProgress {

    public CommentsScrollProgress() {
        collapsedIDs = new HashSet<>();
    }

    public int storyId;
    public int topCommentId;
    public int topCommentOffset;
    public HashSet<Integer> collapsedIDs;

}

File: app/src/main/java/com/simon/harmonichackernews/data/CommentsScrollProgress.kt
package com.simon.harmonichackernews.data

import java.util.HashSet

class CommentsScrollProgress {
    var storyId: Int = 0
    var topCommentId: Int = 0
    var topCommentOffset: Int = 0
    var collapsedIDs: HashSet<Int> = HashSet()
}
